src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -155,80 +179,81 @@ private JPanel getPanelMisc() {
 			outputTabTimeStampLabel = new JLabel(Constant.messages.getString("options.display.timestamp.format.outputtabtimestamps.label"));
 			largeRequestLabel = new JLabel(Constant.messages.getString("view.options.label.largeRequestSize"));
 			largeResponseLabel = new JLabel(Constant.messages.getString("view.options.label.largeResponseSize"));
-					
+			lookAndFeelLabel = new JLabel(Constant.messages.getString("view.options.label.lookandfeel"));
 			outputTabTimeStampExampleLabel = new JLabel(TimeStampUtils.currentDefaultFormattedTimeStamp());
 			showSplashScreenLabel = new JLabel(Constant.messages.getString("view.options.label.showSplashScreen"));
 			
 			int row = 0;
 			displayLabel.setLabelFor(getDisplaySelect());
 			panelMisc.add(displayLabel, 
-					LayoutHelper.getGBC(0, row, 1, 1.0D, new java.awt.Insets(2,2,2,2)));
+					LayoutHelper.getGBC(0, row, 1, 1.0D, new java.awt.Insets(2,1,2,1)));

```

Comment:
Worth reusing the same instance (less changes if this needs to be tweaked again).